src/core/string_map.h

```diff
@@ -147,6 +147,8 @@ class StringMap : public DenseSet {
   void RandomPairs(unsigned int count, std::vector<sds>& keys, std::vector<sds>& vals,
                    bool with_value);
 
+  static sds GetValue(sds key, std::optional<size_t> len = std::nullopt);
```

Comment:
I do not think that passing `len` is justified here: the interface becomes more confusing so that we could save `sdslen()` call which is very fast.

Reply:
fixed